tests/Microsoft.Identity.Test.Unit/AuthorityAliasesTests.cs

```diff
@@ -108,7 +120,18 @@ public async Task AuthorityMigrationTestAsync()
 
                     try
                     {
-                        result = await app
+                        var app3 = PublicClientApplicationBuilder
+                               .Create(TestConstants.ClientId)
+                                     .WithAuthority($"https://{envAlias}/common", true)
```

Comment:
>true [](http://example.com/codeflow?start=82&length=4)

what if .WithAuthority("", false)?

Reply:
Added all 4 combinations of tests now: 

validateAuthority = true | false
isAuthorityValid = true | false